### ISSUE TITLE
Organization of the website

### ISSUE THREAD

**Issue by chewyboba10** [mmaxwu/reunion#3]
Currently in the early stages of development.
Here is the organization idea I have come up with:
- the menu items are for each of the 9 national parks in california linking to a different page
- each page will have wildlife information, greenery, things to do/places to visit, weather, and pictures
- homepage with intro on what the website is and what we hope to accomplish

Please comment if you have any other suggestions

**Comment by JishnuS420** [comment]
I agree with this idea, I think it encapsulates our ideas pretty well and would give the site some good structure and organization. Looking at it from a user's perspective, it would also be a lot easier to navigate through the site and the home page would let the user know what they are looking at.
